tests/yamls.go

```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
       - name: ubuntu
-        image: prateek14/ubuntu:18.04
+        image: quay.io/payesanand/ubuntu:18.04
```

Comment:
can we use the  official ubuntu image here?

Reply:
pause binaries are not present in official images